ingestion/src/metadata/ingestion/bulksink/metadata_usage.py

```diff
@@ -348,9 +350,43 @@ def _get_table_life_cycle_data(
         the query being processed.
         """
         try:
-            # Temporary clean up this method to be implemented following the new schema
-            life_cycle = (  # pylint: disable=unused-variable
-                init_empty_life_cycle_properties()
+            life_cycle = LifeCycle()
+            for create_query in table_usage.sqlQueries:
+                user = None
+                process_user = None
+                if create_query.users:
+                    user = self.metadata.get_entity_reference(
+                        entity=User, fqn=create_query.users[0]
+                    )
+                elif create_query.usedBy:
+                    process_user = create_query.usedBy[0]
+                query_type = get_query_type(create_query=create_query)
+                access_details = AccessDetails(
+                    timestamp=create_query.queryDate.__root__,
+                    accessedBy=user,
+                    accessedByAProcess=process_user,
+                )
+                if query_type == "Created" and (
+                    not life_cycle.created
+                    or life_cycle.created.timestamp.__root__
+                    < access_details.timestamp.__root__
```

Comment:
`or` part is repeated on all the ifs, maybe we could modify it to have it used ones

Reply:
variables after the or are different